src/cli-builder.ts

```diff
@@ -58,6 +59,12 @@ export const buildCli = (): Argv =>
 								defaultConfig.reportsFolder as string),
 							type: 'string',
 						},
+						dockerRunArgs: {
+							describe: 'Extra arguments for the "docker run" command when Docker is enabled. E.g.: ' +
+								'--dockerRunArgs=-e --dockerRunArgs=TEST_VAR=myValue',
+							type: 'string',
+							array: true,
```

Comment:
Due to use in `execa`, I couldn't just allow a string here, it had to be an array. If we want splitting by spaces then we also need to support some escape mechanism for unwanted splits (when a space is part of the argument).